prow/tide/tide.go

```diff
@@ -35,13 +37,20 @@ import (
 	"k8s.io/test-infra/prow/pjutil"
 )
 
+const (
+	statusContext   string = "tide"
+	statusInPool           = "Will consider for merge once all tests are passing."
+	statusNotInPool        = "Not in tide pool."
```

Comment:
This status is going to be meaningless for people who don't know what tide is, or what the concept of the tide pool is.  How we can make this more discoverable or intuitive?

Reply:
Another way of thinking about this... is this telling the author/reviewer there's some action that should be taken?

Reply:
Changing the "Not in tide pool." message to a specific explanation of why the PR isn't mergable (like what the SQ does now) is something we intend to add in another PR: https://github.com/kubernetes/test-infra/pull/6010#discussion_r157597197

Reply:
There's only so much info we can fit into the description here. We also need the link to be something useful, which @cjwagner has done some work on.